# libgda ja.po
# Copyright (C) 1999 Free Software Foundation, Inc.
# Akira TAGOH <tagoh@gnome.gr.jp>, 2000-2001.
#
msgid ""
msgstr ""
"Project-Id-Version: libgda CVS-20010407\n"
"POT-Creation-Date: 2003-01-28 01:43+0100\n"
"PO-Revision-Date: 2001-04-07 12:45+0900\n"
"Last-Translator: Akira TAGOH <tagoh@gnome.gr.jp>\n"
"Language-Team: Japanese <translation@gnome.gr.jp>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: libgda/gda-client.c:300 libgda/gda-client.c:502 libgda/gda-connection.c:218
#, fuzzy, c-format
msgid "Data source %s not found in configuration"
msgstr "データベース '%s' はシステム設定から見つかりませんでした"

#: libgda/gda-client.c:325
#, fuzzy, c-format
msgid "Could not find provider %s in the current setup"
msgstr "クライアントメッセージ数を取得することが出来ませんでした"

#: libgda/gda-client.c:354
#, fuzzy, c-format
msgid "Provider %s does not implement entry function"
msgstr "未実装の機能をプロバイダ %s で開始しています"

#: libgda/gda-client.c:364
#, fuzzy
msgid "Could not create GdaServerProvider object from plugin"
msgstr "GdaServerCommand の割り当てることが出来ませんでした"

#: libgda/gda-client.c:429
msgid "Temporary data source created by libgda. Don't remove it"
msgstr ""

#: libgda/gda-config.c:1072
#, fuzzy, c-format
msgid "Error: %s"
msgstr "エラー"

#. gda_server_recordset_model_set_field_defined_size (recset, 0, 256);
#: libgda/gda-config.c:1322 providers/default/gda-default-provider.c:555
#: providers/default/gda-default-provider.c:575
#: providers/default/gda-default-provider.c:612
#: providers/ldap/gda-ldap-provider.c:327 providers/mdb/gda-mdb-provider.c:457
#: providers/mdb/gda-mdb-provider.c:594 providers/mdb/gda-mdb-provider.c:654
#: providers/mysql/gda-mysql-provider.c:701
#: providers/mysql/gda-mysql-provider.c:889
#: providers/odbc/gda-odbc-provider.c:733
#: providers/odbc/gda-odbc-provider.c:789
#: providers/oracle/gda-oracle-provider.c:1045
#: providers/oracle/gda-oracle-provider.c:1544
msgid "Name"
msgstr ""

#: libgda/gda-config.c:1323
msgid "Provider"
msgstr ""

#: libgda/gda-config.c:1324
#, fuzzy
msgid "Connection string"
msgstr "接続は切れたようです"

#: libgda/gda-config.c:1325 providers/odbc/gda-odbc-provider.c:1142
#: providers/odbc/gda-odbc-provider.c:1298
#: providers/postgres/gda-postgres-provider.c:1106
msgid "Description"
msgstr ""

#: libgda/gda-config.c:1326
#, fuzzy
msgid "Username"
msgstr "ユーザ名"

#: libgda/gda-config.c:1327
#, fuzzy
msgid "Password"
msgstr "パスワード"

#: libgda/gda-data-model.c:686
#, fuzzy, c-format
msgid "Data model %p is not editable"
msgstr "%p は有効なテーブルではありません"

#: libgda/gda-init.c:48
msgid "Attempt to initialize an already initialized client"
msgstr "初期化を試みましたがすでにクライアントは初期化されています"

#: libgda/gda-init.c:59
msgid "libgda needs GModule. Finishing..."
msgstr ""

#: libgda/gda-select.c:315
#, fuzzy, c-format
msgid "Could not parse SQL string '%s'"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: libgda/gda-select.c:319
#, c-format
msgid "SQL command is not a SELECT (is '%s'"
msgstr ""

#: libgda/gda-table.c:286
#, c-format
msgid "There is already a field called %s"
msgstr ""

#: libgda/gda-util.c:144
#, fuzzy, c-format
msgid "Could not parse SQL command '%s'"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: libgda/gda-util.c:177
#, c-format
msgid "Error while reading %s: %s"
msgstr ""

#: libgda/gda-util.c:203
#, fuzzy, c-format
msgid "Could not create file %s"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: libgda/gda-value.c:1650
msgid "TRUE"
msgstr "TRUE"

#: libgda/gda-value.c:1652
msgid "FALSE"
msgstr "FALSE"

#: libgda/gda-value.c:1712
#, c-format
msgid "(GObject of type '%s'"
msgstr ""

#: libgda/gda-value.c:1715
msgid "NULL GObject"
msgstr ""

#: libgda/gda-xml-database.c:235
#, fuzzy, c-format
msgid "Could not load file at %s"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: libgda/gda-xml-database.c:244
#, fuzzy, c-format
msgid "Could not parse file at %s"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: libgda/gda-xml-database.c:254
#, fuzzy, c-format
msgid "Invalid XML database file '%s'"
msgstr "データベース '%s' を選択しました"

#: libgda/gda-xml-database.c:434
#, fuzzy, c-format
msgid "Could not create a XML node from table %s"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: libgda/gda-xml-database.c:452
#, fuzzy
msgid "Could not dump XML file to memory"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: libgda/gda-xml-database.c:545 libgda/gda-xml-database.c:581
#: libgda/gda-xml-database.c:624
#, fuzzy, c-format
msgid "Table %s already exists"
msgstr "テーブル %s はすでに存在しています"

#: libgda/gda-xml-database.c:617
msgid "Invalid node"
msgstr ""

#: libgda/gda-xml-database.c:659
#, c-format
msgid "Duplicated <data> node for table %s"
msgstr ""

#: libgda/gda-xml-database.c:666
msgid "Invalid XML node"
msgstr ""

#: providers/default/gda-default-provider.c:190
#: providers/sqlite/gda-sqlite-provider.c:198
msgid ""
"A full path must be specified on the connection string to open a database."
msgstr ""

#: providers/default/gda-default-provider.c:252
#: providers/default/gda-default-provider.c:267
#: providers/default/gda-default-provider.c:294
msgid "Invalid internal handle"
msgstr ""

#: providers/default/gda-default-provider.c:333
#, fuzzy
msgid "Not Implemented"
msgstr "%s は実装されていません"

#: providers/default/gda-default-provider.c:454
#: providers/mdb/gda-mdb-provider.c:490
#: providers/mysql/gda-mysql-provider.c:1069
#: providers/odbc/gda-odbc-provider.c:914
#: providers/oracle/gda-oracle-provider.c:1102
#: providers/postgres/gda-postgres-provider.c:1351
#: providers/sqlite/gda-sqlite-provider.c:622
#, fuzzy
msgid "Field name"
msgstr "ファイル名"

#: providers/default/gda-default-provider.c:455
#: providers/mdb/gda-mdb-provider.c:491
#: providers/mysql/gda-mysql-provider.c:1070
#: providers/odbc/gda-odbc-provider.c:915
#: providers/oracle/gda-oracle-provider.c:1103
#: providers/postgres/gda-postgres-provider.c:1352
#: providers/sqlite/gda-sqlite-provider.c:623
msgid "Data type"
msgstr ""

#: providers/default/gda-default-provider.c:456
#: providers/mdb/gda-mdb-provider.c:492
#: providers/mysql/gda-mysql-provider.c:1071
#: providers/odbc/gda-odbc-provider.c:916
#: providers/oracle/gda-oracle-provider.c:1104
#: providers/postgres/gda-postgres-provider.c:1353
#: providers/sqlite/gda-sqlite-provider.c:624
msgid "Size"
msgstr ""

#: providers/default/gda-default-provider.c:457
#: providers/mdb/gda-mdb-provider.c:493
#: providers/mysql/gda-mysql-provider.c:1072
#: providers/odbc/gda-odbc-provider.c:917
#: providers/oracle/gda-oracle-provider.c:1105
#: providers/postgres/gda-postgres-provider.c:1354
#: providers/sqlite/gda-sqlite-provider.c:625
#, fuzzy
msgid "Scale"
msgstr "False"

#: providers/default/gda-default-provider.c:458
#: providers/mdb/gda-mdb-provider.c:494
#: providers/mysql/gda-mysql-provider.c:1073
#: providers/odbc/gda-odbc-provider.c:918
#: providers/oracle/gda-oracle-provider.c:1106
#: providers/postgres/gda-postgres-provider.c:1355
#: providers/sqlite/gda-sqlite-provider.c:626
msgid "Not null?"
msgstr ""

#: providers/default/gda-default-provider.c:459
#: providers/mdb/gda-mdb-provider.c:495
#: providers/mysql/gda-mysql-provider.c:1074
#: providers/odbc/gda-odbc-provider.c:919
#: providers/oracle/gda-oracle-provider.c:1107
#: providers/postgres/gda-postgres-provider.c:1356
#: providers/sqlite/gda-sqlite-provider.c:627
msgid "Primary key?"
msgstr ""

#: providers/default/gda-default-provider.c:460
#: providers/mdb/gda-mdb-provider.c:496
#: providers/mysql/gda-mysql-provider.c:1075
#: providers/odbc/gda-odbc-provider.c:920
#: providers/oracle/gda-oracle-provider.c:1108
#: providers/postgres/gda-postgres-provider.c:1357
#: providers/sqlite/gda-sqlite-provider.c:628
msgid "Unique index?"
msgstr ""

#: providers/default/gda-default-provider.c:461
#: providers/mdb/gda-mdb-provider.c:497
#: providers/mysql/gda-mysql-provider.c:1076
#: providers/odbc/gda-odbc-provider.c:921
#: providers/oracle/gda-oracle-provider.c:1109
#: providers/postgres/gda-postgres-provider.c:1358
#: providers/sqlite/gda-sqlite-provider.c:629
msgid "References"
msgstr ""

#: providers/default/gda-default-provider.c:471
#: providers/default/gda-default-provider.c:477
msgid "Invalid argument"
msgstr ""

#: providers/default/gda-default-provider.c:484
#: providers/mdb/gda-mdb-table.c:48
#, fuzzy, c-format
msgid "Table %s not found"
msgstr "データソース %s は見つかりませんでした"

#: providers/default/gda-default-provider.c:509
#, fuzzy
msgid "Could not get description for column"
msgstr "%s へ接続することが出来ませんでした"

#: providers/default/libmain.c:33
msgid "Default"
msgstr ""

#: providers/default/libmain.c:39
msgid "Default provider, based on XML"
msgstr ""

#: providers/firebird/gda-firebird-provider.c:187
msgid "No database specified"
msgstr ""

#: providers/firebird/gda-firebird-provider.c:234
#: providers/firebird/gda-firebird-provider.c:261
#: providers/firebird/gda-firebird-provider.c:279
#: providers/firebird/gda-firebird-provider.c:313
#: providers/firebird/gda-firebird-provider.c:368
#: providers/firebird/gda-firebird-provider.c:413
#: providers/firebird/gda-firebird-provider.c:447
#: providers/firebird/gda-firebird-provider.c:485
#: providers/firebird/gda-firebird-provider.c:539
msgid "Invalid Firebird handle"
msgstr ""

#: providers/firebird/gda-firebird-provider.c:453
#: providers/firebird/gda-firebird-provider.c:491
msgid "Invalid transaction handle"
msgstr ""

#: providers/firebird/libmain.c:39
#, fuzzy
msgid "Provider for FireBird databases"
msgstr "MDB (MS Access)ファイル用のGDAデータソースアクセス"

#: providers/freetds/gda-freetds-provider.c:195
msgid ""
"Connection aborted. You must provide at least a host, username and password "
"using DSN 'QUERY=;USER=;PASSWORD='."
msgstr ""

#: providers/freetds/gda-freetds-provider.c:208
msgid "Error creating message container."
msgstr ""

#: providers/freetds/gda-freetds-provider.c:215
msgid "Error creating error container."
msgstr ""

#: providers/freetds/gda-freetds-provider.c:253
#: providers/freetds/gda-freetds-provider.c:255
#, fuzzy
msgid "Allocating tds context failed."
msgstr "レコードセットの割り当てに失敗しました"

#: providers/freetds/gda-freetds-provider.c:271
#: providers/freetds/gda-freetds-provider.c:273
msgid "Establishing connection failed."
msgstr ""

#: providers/freetds/gda-freetds-provider.c:285
#: providers/freetds/gda-freetds-provider.c:286
#, fuzzy
msgid "Failed getting connection info."
msgstr "再接続を試みています"

#: providers/freetds/gda-freetds-provider.c:766
#: providers/freetds/gda-freetds-provider.c:767
msgid "Query did not succeed in execute_cmd()."
msgstr ""

#: providers/freetds/gda-freetds-provider.c:776
#: providers/freetds/gda-freetds-provider.c:778
msgid "Unexpected result tokens in execute_cmd()."
msgstr ""

#: providers/freetds/gda-freetds-provider.c:786
#: providers/freetds/gda-freetds-provider.c:787
msgid "Unexpected return in execute_cmd()."
msgstr ""

#: providers/freetds/gda-freetds-provider.c:1021
#, c-format
msgid ""
"Msg %d, Level %d, State %d, Server %s, Line %d\n"
"%s\n"
msgstr ""

#: providers/freetds/gda-freetds-provider.c:1040 providers/freetds/utils.c:44
#: providers/oracle/utils.c:71 providers/postgres/utils.c:51
#: providers/sybase/utils.c:171
msgid "Not available"
msgstr "利用できません"

#: providers/freetds/gda-freetds-recordset.c:360
msgid "Error processing result rows.\n"
msgstr ""

#: providers/freetds/gda-freetds-recordset.c:367
msgid "Unexpected freetds return code in tds_process_row_tokens().\n"
msgstr ""

#: providers/freetds/gda-freetds-recordset.c:377
msgid "Error processing results.\n"
msgstr ""

#: providers/freetds/gda-freetds-recordset.c:384
msgid "Unexpected freetds return code in tds_process_result_tokens\n"
msgstr ""

#: providers/freetds/libmain.c:39
#, fuzzy
msgid "Provider for TDS-based databases (using FreeTDS)"
msgstr "MDB (MS Access)ファイル用のGDAデータソースアクセス"

#: providers/freetds/utils.c:40 providers/oracle/utils.c:66
#: providers/postgres/utils.c:46 providers/sybase/utils.c:166
msgid "NO DESCRIPTION"
msgstr ""

#: providers/ibmdb2/gda-ibmdb2-provider.c:141
#: providers/ibmdb2/gda-ibmdb2-provider.c:153
#, fuzzy
msgid "Could not allocate environment handle."
msgstr "ステートメントハンドルを割り当てることが出来ませんでした"

#: providers/ibmdb2/gda-ibmdb2-provider.c:164
#, fuzzy
msgid "Could not set autocommit to off.\n"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: providers/ibmdb2/gda-ibmdb2-provider.c:178
#, fuzzy
msgid "Could not allocate environment handle.\n"
msgstr "ステートメントハンドルを割り当てることが出来ませんでした"

#. GetInfo is needed for obtaining database name
#. is needed for tables schema
#: providers/ibmdb2/gda-ibmdb2-provider.c:190
msgid "SQLGetInfo is unsupported. Hence IBM DB2 Provider will not work.\n"
msgstr ""

#: providers/ibmdb2/gda-ibmdb2-provider.c:204
msgid "SQLGetFunctions gave unexpected return. Connection terminated.\n"
msgstr ""

#: providers/ibmdb2/libmain.c:39
#, fuzzy
msgid "Provider for IBMDB2 databases"
msgstr "MDB (MS Access)ファイル用のGDAデータソースアクセス"

#: providers/ldap/gda-ldap-provider.c:257
#: providers/ldap/gda-ldap-provider.c:490
#: providers/ldap/gda-ldap-provider.c:519
#: providers/ldap/gda-ldap-provider.c:548
#: providers/ldap/gda-ldap-provider.c:587
#: providers/ldap/gda-ldap-recordset.c:58
#: providers/ldap/gda-ldap-recordset.c:176
msgid "Invalid LDAP handle"
msgstr ""

#: providers/ldap/gda-ldap-provider.c:328 providers/mdb/gda-mdb-provider.c:548
#: providers/mdb/gda-mdb-provider.c:595 providers/mdb/gda-mdb-provider.c:655
#: providers/mysql/gda-mysql-provider.c:703
#: providers/mysql/gda-mysql-provider.c:890
#: providers/mysql/gda-mysql-provider.c:972
#: providers/odbc/gda-odbc-provider.c:1032
#: providers/odbc/gda-odbc-provider.c:1141
#: providers/odbc/gda-odbc-provider.c:1255
#: providers/odbc/gda-odbc-provider.c:1297
#: providers/odbc/gda-odbc-provider.c:1349
#: providers/oracle/gda-oracle-provider.c:1544
#: providers/postgres/gda-postgres-provider.c:835
#: providers/postgres/gda-postgres-provider.c:1105
#: providers/postgres/gda-postgres-provider.c:1124
#: providers/postgres/gda-postgres-provider.c:1184
#: providers/postgres/gda-postgres-provider.c:1301
#: providers/postgres/gda-postgres-provider.c:1845
#: providers/sqlite/gda-sqlite-provider.c:752
#: providers/sybase/gda-sybase-provider.c:905
msgid "Owner"
msgstr ""

#: providers/ldap/gda-ldap-provider.c:329 providers/mdb/gda-mdb-provider.c:549
#: providers/mdb/gda-mdb-provider.c:596 providers/mdb/gda-mdb-provider.c:656
#: providers/mysql/gda-mysql-provider.c:704
#: providers/mysql/gda-mysql-provider.c:891
#: providers/mysql/gda-mysql-provider.c:973
#: providers/odbc/gda-odbc-provider.c:1033
#: providers/odbc/gda-odbc-provider.c:1256
#: providers/odbc/gda-odbc-provider.c:1350
#: providers/oracle/gda-oracle-provider.c:1544
#: providers/postgres/gda-postgres-provider.c:836
#: providers/postgres/gda-postgres-provider.c:1125
#: providers/postgres/gda-postgres-provider.c:1185
#: providers/postgres/gda-postgres-provider.c:1302
#: providers/postgres/gda-postgres-provider.c:1846
#: providers/sqlite/gda-sqlite-provider.c:753
#: providers/sybase/gda-sybase-provider.c:907
msgid "Comments"
msgstr ""

#. gda_server_recordset_model_set_field_defined_size (recset, 0, 32);
#: providers/ldap/gda-ldap-provider.c:423
#: providers/mysql/gda-mysql-provider.c:971
#: providers/odbc/gda-odbc-provider.c:1254
#: providers/oracle/gda-oracle-provider.c:1643
#: providers/postgres/gda-postgres-provider.c:1123
#: providers/sqlite/gda-sqlite-provider.c:751
#: providers/sybase/gda-sybase-provider.c:903
msgid "Type"
msgstr ""

#: providers/ldap/gda-ldap-recordset.c:69
#: providers/mysql/gda-mysql-recordset.c:69
#: providers/postgres/gda-postgres-recordset.c:185
#: providers/postgres/gda-postgres-recordset.c:225
#: providers/sqlite/gda-sqlite-recordset.c:165
#: providers/sqlite/gda-sqlite-recordset.c:204
msgid "Row number out of range"
msgstr ""

#: providers/ldap/gda-ldap-recordset.c:283
#: providers/mysql/gda-mysql-recordset.c:288
#: providers/oracle/gda-oracle-recordset.c:466
msgid "Attempt to insert a row with an invalid number of columns"
msgstr ""

#: providers/ldap/gda-ldap-recordset.c:298
#: providers/mysql/gda-mysql-recordset.c:303
#: providers/oracle/gda-oracle-recordset.c:481
#, fuzzy
msgid "Could not retrieve column's information"
msgstr "%s へ接続することが出来ませんでした"

#: providers/ldap/gda-ldap-recordset.c:316
#: providers/mysql/gda-mysql-recordset.c:321
#: providers/oracle/gda-oracle-recordset.c:499
#, fuzzy
msgid "Could not retrieve column's value"
msgstr "サーバメッセージを消去することが出来ませんでした"

#: providers/ldap/libmain.c:42
msgid "Provider for LDAP directory"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:205
msgid "A database FILENAME must be specified in the connection_string"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:214
#: providers/xbase/gda-xbase-provider.c:223
#, fuzzy, c-format
msgid "Could not open file %s"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: providers/mdb/gda-mdb-provider.c:238 providers/mdb/gda-mdb-provider.c:268
#: providers/mdb/gda-mdb-provider.c:290 providers/mdb/gda-mdb-provider.c:342
#: providers/mdb/gda-mdb-provider.c:690 providers/mdb/gda-mdb-provider.c:726
msgid "Invalid MDB handle"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:404 providers/mdb/gda-mdb-provider.c:414
#: providers/mdb/gda-mdb-provider.c:424
msgid "Transactions not supported in MDB provider"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:498
#: providers/mysql/gda-mysql-provider.c:1077
#: providers/odbc/gda-odbc-provider.c:922
#: providers/postgres/gda-postgres-provider.c:1359
#: providers/sqlite/gda-sqlite-provider.c:630
msgid "Default value"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:546
#: providers/odbc/gda-odbc-provider.c:1030
#: providers/postgres/gda-postgres-provider.c:833
msgid "Procedure"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:547
#: providers/mysql/gda-mysql-provider.c:702
msgid "ID"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:550
#: providers/mysql/gda-mysql-provider.c:705
#: providers/odbc/gda-odbc-provider.c:1034
#: providers/postgres/gda-postgres-provider.c:837
msgid "Return type"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:551
msgid "# of args"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:552
#: providers/mysql/gda-mysql-provider.c:706
#: providers/odbc/gda-odbc-provider.c:1036
#: providers/postgres/gda-postgres-provider.c:839
msgid "Args types"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:553
#: providers/odbc/gda-odbc-provider.c:1037
#: providers/odbc/gda-odbc-provider.c:1143
#: providers/odbc/gda-odbc-provider.c:1299
#: providers/odbc/gda-odbc-provider.c:1351
#: providers/postgres/gda-postgres-provider.c:840
#: providers/postgres/gda-postgres-provider.c:1107
#: providers/postgres/gda-postgres-provider.c:1186
#: providers/postgres/gda-postgres-provider.c:1305
#: providers/postgres/gda-postgres-provider.c:1847
msgid "Definition"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:657
msgid "GDA Type"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:659
msgid "Boolean type"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:660
msgid "1-byte integers"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:661
msgid "Double precision values"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:662
msgid "Single precision values"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:663
msgid "32-bit integers"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:664
msgid "64-bit integers"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:665
msgid "Variable length character strings"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:666
msgid "Money amounts"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:667
msgid "OLE object"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:668
msgid "FIXME"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:669
msgid "Date/time value"
msgstr ""

#: providers/mdb/gda-mdb-provider.c:670
msgid "Character strings"
msgstr ""

#. end unsafe
#: providers/mdb/gda-mdb-provider.c:737
#, fuzzy, c-format
msgid "Could not parse '%s' command"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: providers/mdb/libmain.c:36
#, fuzzy
msgid "Provider for Microsoft Access files"
msgstr "MDB (MS Access)ファイル用のGDAデータソースアクセス"

#: providers/mysql/gda-mysql-provider.c:218
#, fuzzy
msgid ""
"You cannot provide a UNIX_SOCKET if you also provide either a HOST or a PORT."
msgstr ""
"%s:%d: UNIX_SOCKET を提供することが出来ません. HOST または PORT のどちらかを"
"提供するなら, UNIX_SOCKET を削除するか, HOST と PORT の両方のオプションを削除"
"してください"

#: providers/mysql/gda-mysql-provider.c:294
#: providers/mysql/gda-mysql-provider.c:1085
#: providers/mysql/gda-mysql-recordset.c:58
#: providers/mysql/gda-mysql-recordset.c:174
msgid "Invalid MySQL handle"
msgstr ""

#: providers/mysql/gda-mysql-provider.c:310
#: providers/mysql/gda-mysql-provider.c:380
#: providers/mysql/gda-mysql-provider.c:403
#: providers/mysql/gda-mysql-provider.c:434
#: providers/mysql/gda-mysql-provider.c:466
#: providers/mysql/gda-mysql-provider.c:534
#: providers/mysql/gda-mysql-provider.c:591
#: providers/mysql/gda-mysql-provider.c:624
msgid "Invalid MYSQL handle"
msgstr ""

#: providers/mysql/gda-mysql-provider.c:539
#: providers/mysql/gda-mysql-provider.c:596
#: providers/mysql/gda-mysql-provider.c:629
#: providers/oracle/gda-oracle-provider.c:822
#: providers/oracle/gda-oracle-provider.c:904
#: providers/oracle/gda-oracle-provider.c:965
#: providers/sqlite/gda-sqlite-provider.c:472
msgid "Transactions are not supported in read-only mode"
msgstr ""

#: providers/mysql/gda-mysql-provider.c:707
msgid "SQL"
msgstr ""

#: providers/mysql/gda-mysql-provider.c:974
#: providers/odbc/gda-odbc-provider.c:1257
#: providers/postgres/gda-postgres-provider.c:1126
#: providers/sqlite/gda-sqlite-provider.c:754
#: providers/sybase/gda-sybase-provider.c:909
msgid "GDA type"
msgstr ""

#: providers/mysql/gda-mysql-provider.c:1094
#: providers/mysql/gda-mysql-provider.c:1102
msgid "Table name is needed but none specified in parameter list"
msgstr ""

#: providers/mysql/libmain.c:41
#, fuzzy
msgid "Provider for MySQL databases"
msgstr "MDB (MS Access)ファイル用のGDAデータソースアクセス"

#: providers/odbc/gda-odbc-provider.c:183
msgid "Unable to SQLAllocEnv()..."
msgstr ""

#: providers/odbc/gda-odbc-provider.c:433
#: providers/odbc/gda-odbc-provider.c:453
#: providers/odbc/gda-odbc-provider.c:483
#: providers/odbc/gda-odbc-provider.c:513
#: providers/odbc/gda-odbc-provider.c:544
#: providers/odbc/gda-odbc-provider.c:574
msgid "Invalid Odbc handle"
msgstr ""

#: providers/odbc/gda-odbc-provider.c:1031
#: providers/postgres/gda-postgres-provider.c:834
#: providers/postgres/gda-postgres-provider.c:1300
msgid "Id"
msgstr ""

#: providers/odbc/gda-odbc-provider.c:1035
#: providers/postgres/gda-postgres-provider.c:838
msgid "Nb args"
msgstr ""

#. Set it here instead of the SQL query to allow i18n
#: providers/odbc/gda-odbc-provider.c:1140
#: providers/postgres/gda-postgres-provider.c:1104
#: providers/postgres/gda-postgres-provider.c:1899
#, fuzzy
msgid "Table"
msgstr "ファイル名"

#: providers/odbc/gda-odbc-provider.c:1296
#: providers/oracle/gda-oracle-provider.c:1750 testing/client.c:129
msgid "Views"
msgstr ""

#. Set it here instead of the SQL query to allow i18n
#: providers/odbc/gda-odbc-provider.c:1348
#: providers/postgres/gda-postgres-provider.c:1844
#: providers/postgres/gda-postgres-provider.c:1900
msgid "Sequence"
msgstr ""

#: providers/odbc/libmain.c:41
#, fuzzy
msgid "Provider for ODBC data sources"
msgstr "ODBC用のGDAデータソースアクセス"

#: providers/oracle/gda-oracle-provider.c:204
#, fuzzy
msgid "Could not initialize Oracle"
msgstr "ct_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:215
#, fuzzy
msgid "Could not initialize the Oracle environment"
msgstr "ct_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:227
#, fuzzy
msgid "Could not allocate the Oracle service handle"
msgstr "cs_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:237
#, fuzzy
msgid "Could not allocate the Oracle error handle"
msgstr "ct_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:249
#, fuzzy
msgid "Could not allocate the Oracle server handle"
msgstr "cs_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:262
#, fuzzy
msgid "Could not allocate the Oracle session handle"
msgstr "ステートメントハンドルを割り当てることが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:292
#, fuzzy
msgid "Could not attach to the Oracle server"
msgstr "cs_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:308
#, fuzzy
msgid "Could not set the Oracle server attribute in the service context"
msgstr "cs_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:324
#, fuzzy
msgid "Could not set the Oracle username attribute"
msgstr "cs_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:340
#, fuzzy
msgid "Could not set the Oracle password attribute"
msgstr "CORBA ファクトリを開くことが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:355
#, fuzzy
msgid "Could not begin the Oracle session"
msgstr "ct_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:373
#, fuzzy
msgid "Could not set the Oracle session attribute in the service context"
msgstr "cs_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:443
#: providers/oracle/gda-oracle-provider.c:475
#: providers/oracle/gda-oracle-provider.c:549
#: providers/oracle/gda-oracle-provider.c:687
#: providers/oracle/gda-oracle-provider.c:710
#: providers/oracle/gda-oracle-provider.c:733
#: providers/oracle/gda-oracle-provider.c:763
#: providers/oracle/gda-oracle-provider.c:817
#: providers/oracle/gda-oracle-provider.c:899
#: providers/oracle/gda-oracle-provider.c:960
#: providers/oracle/gda-oracle-recordset.c:165
#: providers/oracle/gda-oracle-recordset.c:387
#: providers/oracle/gda-oracle-recordset.c:588
msgid "Invalid Oracle handle"
msgstr ""

#: providers/oracle/gda-oracle-provider.c:486
#, fuzzy
msgid "Could not allocate the Oracle statement handle"
msgstr "ステートメントハンドルを割り当てることが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:497
#, fuzzy
msgid "Could not prepare the Oracle statement"
msgstr "ct_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:529
#, fuzzy
msgid "Could not bind the Oracle statement parameter"
msgstr "CORBA ファクトリを開くことが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:574
#, fuzzy
msgid "Could not get the Oracle statement type"
msgstr "CORBA ファクトリを開くことが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:592
#, fuzzy
msgid "Could not execute the Oracle statement"
msgstr "ct_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:605
#, fuzzy
msgid "Could not get the number of columns in the result set"
msgstr "クライアントメッセージ数を取得することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:622
msgid "Could not get the parameter descripter in the result set"
msgstr ""

#: providers/oracle/gda-oracle-provider.c:634
#, fuzzy
msgid "Could not get the column name in the result set"
msgstr "クライアントメッセージ数を取得することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:780
#: providers/oracle/gda-oracle-provider.c:861
#: providers/oracle/gda-oracle-provider.c:916
#: providers/oracle/gda-oracle-provider.c:977
#, fuzzy
msgid "Could not attach the transaction to the service context"
msgstr "cs_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:835
#, fuzzy
msgid "Could not allocate the Oracle transaction handle"
msgstr "ステートメントハンドルを割り当てることが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:847
#, fuzzy
msgid "Could not set the Oracle transaction name"
msgstr "ct_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:872
#, fuzzy
msgid "Could not start the Oracle transaction"
msgstr "CORBA ファクトリを開くことが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:925
#, fuzzy
msgid "Could not prepare the transaction to commit"
msgstr "%s へ接続することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:933
#, fuzzy
msgid "Could not commit the Oracle transaction"
msgstr "CORBA ファクトリを開くことが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:984
#, fuzzy
msgid "Could not rollback the Oracle transaction"
msgstr "ロケール構造体を割り当てることが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:1273
#, fuzzy
msgid "Could not allocate the Oracle describe handle"
msgstr "ステートメントハンドルを割り当てることが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:1286
#, fuzzy
msgid "Could not describe the Oracle table"
msgstr "サーバメッセージを消去することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:1299
#, fuzzy
msgid "Could not get the Oracle parameter handle"
msgstr "cs_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:1312
#, fuzzy
msgid "Could not get the number of columns in the table"
msgstr "サーバメッセージを消去することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:1324
#, fuzzy
msgid "Could not get the column list"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:1353
#, fuzzy
msgid "Could not get the Oracle column handle"
msgstr "ct_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:1366
#, fuzzy
msgid "Could not get the Oracle field name"
msgstr "ct_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:1387
#, fuzzy
msgid "Could not get the Oracle field data type"
msgstr "CORBA ファクトリを開くことが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:1405
#, fuzzy
msgid "Could not get the Oracle field defined size"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:1422
#, fuzzy
msgid "Could not get the Oracle field scale"
msgstr "クライアントメッセージ数を取得することが出来ませんでした"

#: providers/oracle/gda-oracle-provider.c:1439
msgid "Could not get the Oracle field nullable attribute"
msgstr ""

#. Set it here instead of the SQL query to allow i18n
#: providers/oracle/gda-oracle-provider.c:1505
#: providers/postgres/gda-postgres-provider.c:1803 testing/client.c:132
#, fuzzy
msgid "Users"
msgstr "ユーザ名"

#. Set it here instead of the SQL query to allow i18n
#: providers/oracle/gda-oracle-provider.c:1731
#: providers/postgres/gda-postgres-provider.c:1223 testing/client.c:130
msgid "Indexes"
msgstr ""

#: providers/oracle/gda-oracle-provider.c:1735
msgid "Procedures"
msgstr ""

#: providers/oracle/gda-oracle-provider.c:1739 testing/client.c:133
msgid "Sequences"
msgstr ""

#. Set it here instead of the SQL query to allow i18n
#: providers/oracle/gda-oracle-provider.c:1746
#: providers/postgres/gda-postgres-provider.c:1340 testing/client.c:134
msgid "Triggers"
msgstr ""

#: providers/oracle/gda-oracle-recordset.c:72
#, fuzzy
msgid "Could not get the Oracle parameter"
msgstr "ct_diag() を初期化することが出来ませんでした"

#: providers/oracle/gda-oracle-recordset.c:82
#, fuzzy
msgid "Could not get the parameter defined size"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: providers/oracle/gda-oracle-recordset.c:95
#, fuzzy
msgid "Could not get the parameter data type"
msgstr "フィールドを割り当てることが出来ませんでした"

#: providers/oracle/gda-oracle-recordset.c:126
#, fuzzy
msgid "Could not define by position"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: providers/oracle/gda-oracle-recordset.c:171
#: providers/postgres/gda-postgres-recordset.c:264
msgid "Column out of range"
msgstr ""

#: providers/oracle/gda-oracle-recordset.c:183
#, fuzzy
msgid "Could not get the Oracle parameter information"
msgstr "%s へ接続することが出来ませんでした"

#: providers/oracle/gda-oracle-recordset.c:193
#, fuzzy
msgid "Could not get the Oracle column name"
msgstr "サーバメッセージを消去することが出来ませんでした"

#: providers/oracle/gda-oracle-recordset.c:208
#, fuzzy
msgid "Could not get the Oracle column data type"
msgstr "CORBA ファクトリを開くことが出来ませんでした"

#: providers/oracle/gda-oracle-recordset.c:220
#, fuzzy
msgid "Could not get the Oracle column scale"
msgstr "サーバメッセージを消去することが出来ませんでした"

#: providers/oracle/gda-oracle-recordset.c:232
#, fuzzy
msgid "Could not get the Oracle column nullable attribute"
msgstr "CORBA ファクトリを開くことが出来ませんでした"

#: providers/oracle/gda-oracle-recordset.c:244
#, fuzzy
msgid "Could not get the Oracle defined size"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: providers/oracle/gda-oracle-recordset.c:352
#, fuzzy
msgid "Could not fetch a row"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: providers/oracle/libmain.c:34
#, fuzzy
msgid "Provider for Oracle databases"
msgstr "MDB (MS Access)ファイル用のGDAデータソースアクセス"

#: providers/postgres/gda-postgres-provider.c:553
#: providers/postgres/gda-postgres-provider.c:570
#: providers/postgres/gda-postgres-provider.c:641
#: providers/postgres/gda-postgres-provider.c:776
#: providers/postgres/gda-postgres-recordset.c:176
#: providers/postgres/gda-postgres-recordset.c:216
#: providers/postgres/gda-postgres-recordset.c:258
msgid "Invalid PostgreSQL handle"
msgstr ""

#. Set it here instead of the SQL query to allow i18n
#: providers/postgres/gda-postgres-provider.c:1183
msgid "View"
msgstr ""

#. Set it here instead of the SQL query to allow i18n
#: providers/postgres/gda-postgres-provider.c:1299
msgid "Aggregate"
msgstr ""

#: providers/postgres/gda-postgres-provider.c:1303
msgid "OutType"
msgstr ""

#: providers/postgres/gda-postgres-provider.c:1304
msgid "InType"
msgstr ""

#. Set it here instead of the SQL query to allow i18n
#: providers/postgres/gda-postgres-provider.c:1768 testing/client.c:125
#, fuzzy
msgid "Databases"
msgstr "データベース名"

#: providers/postgres/gda-postgres-recordset.c:231
#: providers/sqlite/gda-sqlite-recordset.c:210
#: providers/sqlite/gda-sqlite-recordset.c:242
msgid "Column number out of range"
msgstr ""

#: providers/postgres/gda-postgres-recordset.c:354
#, c-format
msgid "Tuples:\"%s\""
msgstr ""

#: providers/postgres/libmain.c:41
#, fuzzy
msgid "Provider for PostgreSQL databases"
msgstr "MDB (MS Access)ファイル用のGDAデータソースアクセス"

#: providers/sqlite/gda-sqlite-provider.c:220
msgid "Could not set empty_result_callbacks SQLite option"
msgstr ""

#: providers/sqlite/gda-sqlite-provider.c:243
#: providers/sqlite/gda-sqlite-provider.c:343
#: providers/sqlite/gda-sqlite-provider.c:554
#: providers/sqlite/gda-sqlite-provider.c:719
#: providers/sqlite/gda-sqlite-recordset.c:156
#: providers/sqlite/gda-sqlite-recordset.c:195
#: providers/sqlite/gda-sqlite-recordset.c:236
msgid "Invalid SQLite handle"
msgstr ""

#: providers/sqlite/gda-sqlite-provider.c:265
msgid "Invalid SQLITE handle"
msgstr ""

#: providers/sqlite/gda-sqlite-provider.c:361
#: providers/sqlite/gda-sqlite-provider.c:376
msgid "Only one database per connection is allowed"
msgstr ""

#: providers/sqlite/gda-sqlite-provider.c:390
msgid "To remove a SQLite database you should remove the database file by hand"
msgstr ""

#: providers/sqlite/gda-sqlite-provider.c:664
#, fuzzy
msgid "Could not retrieve information for field"
msgstr "%s へ接続することが出来ませんでした"

#: providers/sqlite/libmain.c:41
#, fuzzy
msgid "Provider for SQLite databases"
msgstr "MDB (MS Access)ファイル用のGDAデータソースアクセス"

#. the logic to connect to the database
#: providers/sybase/gda-sybase-provider.c:250
msgid "about to open connection"
msgstr ""

#: providers/sybase/gda-sybase-provider.c:276
msgid "Out of memory. Allocating connection structure failed."
msgstr ""

#: providers/sybase/gda-sybase-provider.c:288
#, fuzzy
msgid "Context allocated."
msgstr "フィールドを割り当てることが出来ませんでした"

#: providers/sybase/gda-sybase-provider.c:295
#, fuzzy
msgid "Client library initialized."
msgstr "ct_diag() を初期化しました"

#: providers/sybase/gda-sybase-provider.c:321
#, c-format
msgid "Locale set to '%s'."
msgstr ""

#: providers/sybase/gda-sybase-provider.c:336
#, fuzzy
msgid "Could not initialize cslib callback"
msgstr "ct_diag() を初期化することが出来ませんでした"

#: providers/sybase/gda-sybase-provider.c:340
msgid "CS-lib callback initialized."
msgstr ""

#: providers/sybase/gda-sybase-provider.c:346
#, fuzzy
msgid "Could not initialize ct-client callback"
msgstr "ct_diag() を初期化することが出来ませんでした"

#: providers/sybase/gda-sybase-provider.c:350
msgid "CT-Client callback initialized."
msgstr ""

#: providers/sybase/gda-sybase-provider.c:357
#, fuzzy
msgid "Could not initialize ct-server callback"
msgstr "ct_diag() を初期化することが出来ませんでした"

#: providers/sybase/gda-sybase-provider.c:361
msgid "CT-Server callback initialized."
msgstr ""

#: providers/sybase/gda-sybase-provider.c:369
#, fuzzy
msgid "Connection allocated."
msgstr "接続は切れたようです"

#: providers/sybase/gda-sybase-provider.c:409
msgid "Appname set."
msgstr ""

#: providers/sybase/gda-sybase-provider.c:422
#, fuzzy
msgid "Username set."
msgstr "ユーザ名"

#: providers/sybase/gda-sybase-provider.c:436
#, fuzzy
msgid "Password set."
msgstr "パスワード"

#: providers/sybase/gda-sybase-provider.c:448
msgid "Empty password set."
msgstr ""

#: providers/sybase/gda-sybase-provider.c:458
#, fuzzy
msgid "Connected."
msgstr "%s に接続完了"

#: providers/sybase/gda-sybase-provider.c:476
msgid "Finally connected."
msgstr ""

#: providers/sybase/gda-sybase-provider.c:610
msgid ""
"Cmd structure already in use. Command will not be executed because of "
"blocked connection."
msgstr ""

#: providers/sybase/gda-sybase-provider.c:624
#, fuzzy, c-format
msgid "Failed allocating a command structure in %s()"
msgstr "コマンド構造体の充填に失敗しました\n"

#: providers/sybase/gda-sybase-provider.c:635
#, fuzzy
msgid "Could not prepare command structure."
msgstr "コマンド構造体を割り当てることが出来ませんでした"

#: providers/sybase/gda-sybase-provider.c:644
#, fuzzy
msgid "Sending sql-command failed."
msgstr "%s: コマンドの送信は失敗しました"

#: providers/sybase/gda-sybase-provider.c:713
#, fuzzy
msgid "Failed dropping command structure."
msgstr "コマンド構造体の充填に失敗しました\n"

#: providers/sybase/gda-sybase-provider.c:743
#, fuzzy, c-format
msgid "Command structure already in use. %s failed."
msgstr "コマンド構造体の開放に失敗しました"

#: providers/sybase/gda-sybase-provider.c:751
#: providers/sybase/gda-sybase-provider.c:855
#, fuzzy
msgid "Could not allocate command structure."
msgstr "コマンド構造体を割り当てることが出来ませんでした"

#: providers/sybase/gda-sybase-provider.c:759
#, fuzzy, c-format
msgid "Could not prepare command structure with %s."
msgstr "コマンド構造体を割り当てることが出来ませんでした"

#: providers/sybase/gda-sybase-provider.c:768
#, fuzzy
msgid "Sending command failed."
msgstr "%s: コマンドの送信は失敗しました"

#: providers/sybase/gda-sybase-provider.c:792
#: providers/sybase/gda-sybase-provider.c:808
#: providers/sybase/gda-sybase-provider.c:818
#, fuzzy, c-format
msgid "%s: %s failed"
msgstr "%s: ユーザコマンドは失敗しました"

#: providers/sybase/gda-sybase-provider.c:853
#, fuzzy, c-format
msgid ""
"Connection already in use. Could not allocate command structure in %s()."
msgstr "コマンド構造体を割り当てることが出来ませんでした"

#: providers/sybase/gda-sybase-provider.c:866
#, fuzzy, c-format
msgid "Failed dropping command structure in %s()."
msgstr "コマンド構造体の充填に失敗しました\n"

#: providers/sybase/gda-sybase-recordset.c:314
#, fuzzy
msgid "Could not allocate datamodel. No results will be returned."
msgstr "コマンド構造体を割り当てることが出来ませんでした"

#: providers/sybase/gda-sybase-recordset.c:317
#, fuzzy
msgid "ct_cancel() failed."
msgstr "%s: ct_command は失敗しました"

#: providers/sybase/gda-sybase-recordset.c:329
#, c-format
msgid "%s failed while processing a row result."
msgstr ""

#: providers/sybase/gda-sybase-recordset.c:336
#, fuzzy, c-format
msgid "%s returned <= 0 columns."
msgstr "%d カラムをカウントしました"

#: providers/sybase/gda-sybase-recordset.c:356
#, fuzzy
msgid "Could not allocate structure for column metainformation."
msgstr "%s へ接続することが出来ませんでした"

#: providers/sybase/gda-sybase-recordset.c:360
#, fuzzy, c-format
msgid "Could not call %s while processing row resultset."
msgstr "クライアントメッセージ数を取得することが出来ませんでした"

#: providers/sybase/gda-sybase-recordset.c:378
#: providers/sybase/gda-sybase-recordset.c:397
#, fuzzy, c-format
msgid "Could not run %s on column %d"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: providers/sybase/gda-sybase-recordset.c:386
#, fuzzy, c-format
msgid "Could not allocate data placeholder for column %d"
msgstr "データホルダーを割り当てることが出来ませんでした"

#: providers/sybase/gda-sybase-recordset.c:415
#, fuzzy, c-format
msgid "Could not run %s to cancel processing row resultset."
msgstr "クライアントメッセージ数を取得することが出来ませんでした"

#: providers/sybase/gda-sybase-recordset.c:430
#, c-format
msgid "%s failed on row %d"
msgstr ""

#: providers/sybase/gda-sybase-recordset.c:456
#, c-format
msgid "%s terminated with unexpected return code."
msgstr ""

#: providers/sybase/gda-sybase-recordset.c:530
#, fuzzy, c-format
msgid "%s returned %s"
msgstr "ct_results() は %x を返しました"

#: providers/sybase/gda-sybase-types.c:94
#: providers/sybase/gda-sybase-types.c:97
msgid "Attempt setting a nonnullable field to null."
msgstr ""

#: providers/sybase/gda-sybase-types.c:208
#, c-format
msgid "cslib cannot convert type %d"
msgstr ""

#: providers/sybase/gda-sybase-types.c:223
#, c-format
msgid "data conversion failed for type %d"
msgstr ""

#: providers/sybase/libmain.c:40
#, fuzzy
msgid "Provider for sybase databases (using OpenClient)"
msgstr "MDB (MS Access)ファイル用のGDAデータソースアクセス"

#: providers/sybase/main.c:51
#, fuzzy
msgid "Could not initiate Sybase component factory"
msgstr "CORBA ファクトリを開くことが出来ませんでした"

#: providers/sybase/utils.c:37
msgid "Call: csmsg callback"
msgstr ""

#: providers/sybase/utils.c:42
#, c-format
msgid ""
"CS-Library error:\n"
"\tseverity(%ld) layer(%ld) origin (%ld) number(%ld)\n"
"\t%s"
msgstr ""

#: providers/sybase/utils.c:49
#, fuzzy, c-format
msgid "OS error: %s"
msgstr "致命的なエラー: %s\n"

#: providers/sybase/utils.c:59
msgid "Call: Client callback."
msgstr ""

#: providers/sybase/utils.c:64
#, c-format
msgid ""
"CT-Client error:\n"
"\tseverity(%ld) layer(%ld) origin (%ld) number(%ld)\n"
"\t%s"
msgstr ""

#: providers/sybase/utils.c:79
msgid "Call: server callback"
msgstr ""

#: providers/sybase/utils.c:84
#, c-format
msgid ""
"CT-Server message:\n"
"\tnumber(%ld) severity(%ld) state(%ld) line(%ld)"
msgstr ""

#: providers/sybase/utils.c:89
#, fuzzy, c-format
msgid "\tServer name: %s\n"
msgstr "ユーザ名"

#: providers/sybase/utils.c:92
#, fuzzy, c-format
msgid "\tProcedure name: %s\n"
msgstr "ユーザ名"

#: providers/sybase/utils.c:130
msgid "ct_diag() failed determining # of client messages."
msgstr ""

#: providers/sybase/utils.c:138
#, fuzzy, c-format
msgid "%d messages fetched."
msgstr "%d 個の ctlib メッセージを取得しました"

#: providers/sybase/utils.c:152
msgid "Operation not possible, connection busy."
msgstr ""

#: providers/xbase/gda-xbase-provider.c:185
msgid "Either FILENAME or DIRECTORY can be specified, but not both or neither"
msgstr ""

#: providers/xbase/gda-xbase-provider.c:232
msgid "Either FILENAME or DIRECTORY must be specified in the connection string"
msgstr ""

#: providers/xbase/gda-xbase-provider.c:261
#: providers/xbase/gda-xbase-provider.c:379
#, fuzzy
msgid "Invalid Xbase handle"
msgstr "データベース '%s' を選択しました"

#: providers/xbase/libmain.c:39
#, fuzzy
msgid "Provider for Xbase databases"
msgstr "MDB (MS Access)ファイル用のGDAデータソースアクセス"

#: report/libgda-report/gda-report-document.c:161
#, fuzzy
msgid "Could not parse XML document"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: report/libgda-report/gda-report-document.c:171
msgid "XML document is not valid"
msgstr ""

#: report/libgda-report/gda-report-document.c:195
#, fuzzy, c-format
msgid "Could not get file from %s"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: report/libgda-report/gda-report-item.c:276
msgid "Error setting parent -> child relation"
msgstr ""

#: report/libgda-report/gda-report-item.c:375
#, c-format
msgid "Error setting value %s to attribute %s of item %s"
msgstr ""

#: report/libgda-report/gda-report-valid.c:130
#, fuzzy, c-format
msgid "could not get DTD from %s"
msgstr "'%s' にロケールを設定することが出来ませんでした"

#: report/libgda-report/gda-report-valid.c:231
#, c-format
msgid "Error validating element %s"
msgstr ""

#: report/libgda-report/gda-report-valid.c:269
#, c-format
msgid "No declaration for attribute %s of element %s\n"
msgstr ""

#: report/libgda-report/gda-report-valid.c:283
#, c-format
msgid "Value \"%s\" for attribute %s of %s is not among the enumerated set\n"
msgstr ""

#: testing/client.c:41
msgid "\t\tNONE\n"
msgstr ""

#: testing/client.c:81
#, c-format
msgid "** ERROR: could not open connection to %s\n"
msgstr ""

#. show provider features
#: testing/client.c:86
msgid "\tProvider capabilities...\n"
msgstr ""

#: testing/client.c:87
#, c-format
msgid "\t\tAggregates: %s\n"
msgstr ""

#: testing/client.c:89 testing/client.c:92 testing/client.c:95
#: testing/client.c:98 testing/client.c:101 testing/client.c:104
#: testing/client.c:107 testing/client.c:110 testing/client.c:113
#: testing/client.c:116 testing/client.c:119
msgid "Supported"
msgstr ""

#: testing/client.c:89 testing/client.c:92 testing/client.c:95
#: testing/client.c:98 testing/client.c:101 testing/client.c:104
#: testing/client.c:107 testing/client.c:110 testing/client.c:113
#: testing/client.c:116 testing/client.c:119
msgid "Not supported"
msgstr ""

#: testing/client.c:90
#, c-format
msgid "\t\tIndexes: %s\n"
msgstr ""

#: testing/client.c:93
#, c-format
msgid "\t\tNamespaces: %s\n"
msgstr ""

#: testing/client.c:96
#, c-format
msgid "\t\tProcedures: %s\n"
msgstr ""

#: testing/client.c:99
#, c-format
msgid "\t\tSequences: %s\n"
msgstr ""

#: testing/client.c:102
#, c-format
msgid "\t\tSQL: %s\n"
msgstr ""

#: testing/client.c:105
#, c-format
msgid "\t\tTransactions: %s\n"
msgstr ""

#: testing/client.c:108
#, c-format
msgid "\t\tTriggers: %s\n"
msgstr ""

#: testing/client.c:111
#, c-format
msgid "\t\tUsers: %s\n"
msgstr ""

#: testing/client.c:114
#, c-format
msgid "\t\tViews: %s\n"
msgstr ""

#: testing/client.c:117
#, c-format
msgid "\t\tXML queries: %s\n"
msgstr ""

#. show connection schemas
#: testing/client.c:122
#, c-format
msgid ""
"\n"
"\tProvider reports version: %s\n"
"\n"
msgstr ""

#: testing/client.c:124
msgid "Namespaces"
msgstr ""

#: testing/client.c:126
msgid "Stored procedures"
msgstr ""

#: testing/client.c:127
#, fuzzy
msgid "Connection Tables"
msgstr "接続は切れたようです"

#: testing/client.c:128
msgid "Available types"
msgstr ""

#: testing/client.c:131
msgid "Aggregates"
msgstr ""

#: testing/client.c:135
msgid "Languages"
msgstr ""

#: testing/client.c:140 testing/client.c:148
msgid "\tStarting transaction..."
msgstr ""

#: testing/client.c:142 testing/client.c:146 testing/client.c:150
#: testing/client.c:154
#, fuzzy
msgid "Error"
msgstr "エラー"

#: testing/client.c:144
msgid "\tFinishing transaction..."
msgstr ""

#: testing/client.c:152
msgid "\tRolling back transaction..."
msgstr ""

#: testing/client.c:180
msgid "Testing GDA client API"
msgstr ""

#: testing/client.c:189
msgid "** ERROR: gda_config_get_data_source_list returned a NULL item\n"
msgstr ""

#: testing/client.c:193
#, fuzzy, c-format
msgid " Data source = %s, User = %s\n"
msgstr "データソース %s は見つかりませんでした"

#: testing/config.c:84
msgid "** ERROR: gda_config_get_provider_list returned a NULL item\n"
msgstr ""

#: testing/config.c:88
#, c-format
msgid " Provider = %s\n"
msgstr ""

#: testing/config.c:89
#, c-format
msgid "\tlocation = %s\n"
msgstr ""

#: testing/config.c:90
#, c-format
msgid "\tdescription = %s\n"
msgstr ""

#: testing/config.c:92
msgid "\tgda_params ="
msgstr ""

#: report/libgda-report/gda-report-item-pagefooter.c:171
#: report/libgda-report/gda-report-item-pageheader.c:160
#: report/libgda-report/gda-report-item-reportfooter.c:172
#: report/libgda-report/gda-report-item-reportheader.c:171
#, c-format
msgid "An element with ID %s already exists in the report"
msgstr ""

#: report/libgda-report/gda-report-item-pagefooter.c:196
#: report/libgda-report/gda-report-item-pageheader.c:185
#: report/libgda-report/gda-report-item-reportfooter.c:197
#: report/libgda-report/gda-report-item-reportheader.c:196
#, c-format
msgid "Item with ID %s is not a label"
msgstr ""

#: report/libgda-report/gda-report-item-pagefooter.c:224
#: report/libgda-report/gda-report-item-pageheader.c:214
#: report/libgda-report/gda-report-item-reportfooter.c:225
#: report/libgda-report/gda-report-item-reportheader.c:224
#, c-format
msgid "Item with ID %s is not a repfield"
msgstr ""

#, fuzzy
#~ msgid "MESSAGE: %s"
#~ msgstr "メッセージ: "

#, fuzzy
#~ msgid "ERROR: %s"
#~ msgstr "エラー: "

#, fuzzy
#~ msgid "Field Name"
#~ msgstr "ファイル名"

#, fuzzy
#~ msgid "GDA provider for IBMDB2"
#~ msgstr "MDB (MS Access)ファイル用のGDAデータソースアクセス"

#, fuzzy
#~ msgid "Could not load %s provider"
#~ msgstr "CORBA ファクトリを開くことが出来ませんでした"

#, fuzzy
#~ msgid "Could not load report file at %s"
#~ msgstr "'%s' にロケールを設定することが出来ませんでした"

#, fuzzy
#~ msgid "Could not parse report file at %s"
#~ msgstr "'%s' にロケールを設定することが出来ませんでした"

#, fuzzy
#~ msgid "Invalid report file '%s'"
#~ msgstr "データベース '%s' を選択しました"

#, fuzzy
#~ msgid "An error occurred during fetch"
#~ msgstr "コマンド実行中にエラーが発生しました"

#, fuzzy
#~ msgid "CORBA exception: %s"
#~ msgstr "CORBA ユーザ例外: %s"

#, fuzzy
#~ msgid "Could not create global GDA client"
#~ msgstr "%s へ接続することが出来ませんでした"

#, fuzzy
#~ msgid "GDA Report engine"
#~ msgstr "GDA分配レポートエンジン"

#, fuzzy
#~ msgid "Could not parse XML report"
#~ msgstr "'%s' にロケールを設定することが出来ませんでした"

#, fuzzy
#~ msgid "Could not activate report engine"
#~ msgstr "レポートエンジンを始動させることが出来ませんでした"

#, fuzzy
#~ msgid "Could not create BonoboStreamMem object"
#~ msgstr "%s へ接続することが出来ませんでした"

#, fuzzy
#~ msgid "Could not initiate Report component factory"
#~ msgstr "メールプロバイダの実装を登録することが出来ませんでした"

#, fuzzy
#~ msgid "Could not activate object %s"
#~ msgstr "レポートエンジンを始動させることが出来ませんでした"

#, fuzzy
#~ msgid "Could not query CORBA components"
#~ msgstr "CORBA ファクトリを開くことが出来ませんでした"

#~ msgid "%s: An Error occured in the CORBA system."
#~ msgstr "%s: CORBA システムでエラーが発生しました"

#, fuzzy
#~ msgid "Could not initialize Bonobo"
#~ msgstr "ct_diag() を初期化することが出来ませんでした"

#, fuzzy
#~ msgid "Could not notify client about action %d"
#~ msgstr "クライアントメッセージ番号 %d を取得することが出来ませんでした"

#, fuzzy
#~ msgid "Default component factory"
#~ msgstr "%s へ接続することが出来ませんでした"

#, fuzzy
#~ msgid "GDA Default Datasource Access"
#~ msgstr "ODBC用のGDAデータソースアクセス"

#, fuzzy
#~ msgid "Could not initiate Default component factory"
#~ msgstr "メールプロバイダの実装を登録することが出来ませんでした"

#~ msgid "GDA Datasource Access for MySQL"
#~ msgstr "MySQL用のGDAデータソースアクセス"

#, fuzzy
#~ msgid "Could not initiate MySQL component factory"
#~ msgstr "MySQL プロバイダの実装を登録することが出来ませんでした"

#~ msgid "GDA Datasource Access for ODBC"
#~ msgstr "ODBC用のGDAデータソースアクセス"

#, fuzzy
#~ msgid "ODBC component factory"
#~ msgstr "%s へ接続することが出来ませんでした"

#, fuzzy
#~ msgid "Could not initiate ODBC component factory"
#~ msgstr "CORBA ファクトリを開くことが出来ませんでした"

#~ msgid "GDA Datasource Access for Oracle"
#~ msgstr "Oracle用のGDAデータソースアクセス"

#, fuzzy
#~ msgid "Oracle component factory"
#~ msgstr "%s へ接続することが出来ませんでした"

#~ msgid "GDA Datasource Access for Postgres"
#~ msgstr "Postgres用のGDAデータソースアクセス"

#, fuzzy
#~ msgid "Postgres component factory"
#~ msgstr "%s へ接続することが出来ませんでした"

#, fuzzy
#~ msgid "Could not initiate PostgreSQL component factory"
#~ msgstr "MySQL プロバイダの実装を登録することが出来ませんでした"

#, fuzzy
#~ msgid "NULL"
#~ msgstr "<NULL>"

#, fuzzy
#~ msgid "Unknown type"
#~ msgstr "未知の ORACLE タイプ %ld です"

#, fuzzy
#~ msgid "SQLite component factory"
#~ msgstr "%s へ接続することが出来ませんでした"

#, fuzzy
#~ msgid "Could not initiate Sqlite component factory"
#~ msgstr "CORBA ファクトリを開くことが出来ませんでした"

#~ msgid "GDA Datasource Access for Sybase"
#~ msgstr "Sybase用のGDAデータソースアクセス"

#, fuzzy
#~ msgid "Client Message: "
#~ msgstr "クライアントメッセージ:"

#, fuzzy
#~ msgid "Informational Client  Message: "
#~ msgstr "クライアントメッセージ:"

#, fuzzy
#~ msgid "Server Message: "
#~ msgstr "サーバメッセージ:"

#, fuzzy
#~ msgid "Informational Server  Message: "
#~ msgstr "サーバメッセージ:"

#~ msgid "CORBA System exception: %s"
#~ msgstr "CORBA システム例外: %s"

#~ msgid "True"
#~ msgstr "True"

#, fuzzy
#~ msgid "[CORBA User Exception]"
#~ msgstr "CORBA ユーザ例外: %s"

#~ msgid "<Unknown GDA Type(NULL)>"
#~ msgstr "<未知の GDA タイプ(NULL)>"

#~ msgid "Saving log...x"
#~ msgstr "ログを保存中...x"

#~ msgid "Could not get pointer to server implementation"
#~ msgstr "サーバの実装へのポインタを取得することが出来ませんでした"

#~ msgid "Received signal %d, shutting down"
#~ msgstr "シグナル %d を受信しました, 終了します"

#~ msgid "Received signal %d. Ignoring"
#~ msgstr "シグナル %d を受信しました, 無視します"

#~ msgid "Attempt to initialize an already initialized provider"
#~ msgstr "初期化を試みましたがすでにプロバイダは初期化されています"

#~ msgid "CORBA system exception %s"
#~ msgstr "CORBA システム例外 %s"

#~ msgid "GDA Datasource Access for LDAP"
#~ msgstr "LDAP用のGDAデータソースアクセス"

#~ msgid "error '%s' at %s"
#~ msgstr "エラー '%s' が %s で発生しました"

#~ msgid "Could not register LDAP provider implementation"
#~ msgstr "LDAP プロバイダの実装を登録することが出来ませんでした"

#~ msgid "GDA Datasource Access for Interbase"
#~ msgstr "Interbase用のGDAデータソースアクセス"

#~ msgid "Error code %ld"
#~ msgstr "エラーコード %ld"

#~ msgid "Could not register INTERBASE provider implementation"
#~ msgstr "INTERBASE プロバイダの実装を登録することが出来ませんでした"

#~ msgid "GDA Datasource Access for Mail"
#~ msgstr "Mail用のGDAデータソースアクセス"

#~ msgid "Unhandled SCHEMA_QTYPE %d"
#~ msgstr "SCHEMA_QTYPE %d は扱えません"

#~ msgid "Could not register Mail provider implementation"
#~ msgstr "メールプロバイダの実装を登録することが出来ませんでした"

#~ msgid "executing command '%s'"
#~ msgstr "コマンド '%s' を実行しています"

#~ msgid "info '%s'"
#~ msgstr "情報 '%s'"

#~ msgid ""
#~ "%s:%d: You cannot provide a UNIX_SOCKET if you also provide either a HOST "
#~ "or a PORT. Please remove the UNIX_SOCKET, or remove both the HOST and "
#~ "PORT options"
#~ msgstr ""
#~ "%s:%d: UNIX_SOCKET を提供することが出来ません. HOST または PORT のどちらか"
#~ "を提供するなら, UNIX_SOCKET を削除するか, HOST と PORT の両方のオプション"
#~ "を削除してください"

#~ msgid "%s:%d: You must  provide the filename with the MS Access database"
#~ msgstr "%s:%d: MS Access データベースのファイル名を提供しなければなりません"

#~ msgid "Opening connection with filename=%s, user=%s, password=%s"
#~ msgstr "ファイル名=%s, ユーザ=%s, パスワード=%s で接続を開いています"

#~ msgid "mysql_recset is NULL"
#~ msgstr "mysql_recset は NULL です"

#~ msgid "No table name given for SCHEMA_COLS"
#~ msgstr "SCHEMA_COLS のためのテーブル名が与えられていません"

#~ msgid ""
#~ "Opening connection with user=%s, password=%s, host=%s, port=%s, "
#~ "unix_socket=%s"
#~ msgstr ""
#~ "ユーザ=%s, パスワード=%s, ホスト=%s, ポート=%s, unix ソケット=%s で接続を"
#~ "開いています"

#~ msgid "gda_odbc_connection_open_schema: Unhandled SCHEMA_QTYPE %d\n"
#~ msgstr "gda_odbc_connection_open_schema: SCHEMA_QTYPE %d は扱えません\n"

#~ msgid "no text"
#~ msgstr "テキストがありません"

#~ msgid "Could not register ODBC provider implementation"
#~ msgstr "ODBC プロバイダの実装を登録することが出来ませんでした"

#~ msgid "command object being created without associated connection"
#~ msgstr "コマンドオブジェクトは接続なしで作成しました"

#~ msgid "Executing command '%s'"
#~ msgstr "コマンド '%s' を実行しています"

#~ msgid "error in OCIHandleAlloc() HTYPE_SVCCTX"
#~ msgstr "OCIHandleAlloc() HTYPE_SVCCTX でエラーが発生しました"

#~ msgid "error in OCIHandleAlloc() HTYPE_SERVER"
#~ msgstr "OCIHandleAlloc() HTYPE_SERVER でエラーが発生しました"

#~ msgid "error in OCIHandleAlloc() HTYPE_ERROR"
#~ msgstr "OCIHandleAlloc() HTYPE_ERROR でエラーが発生しました"

#~ msgid "Unknown ORACLE type %ld"
#~ msgstr "未知の ORACLE タイプ %ld です"

#~ msgid "Unknown ORACLE data type in fill_field_values"
#~ msgstr "fill_field_values に未知の ORACLE データタイプがあります"

#~ msgid "recordset object being created without associated connection"
#~ msgstr "レコードセットオブジェクトは接続なしで作成しました"

#~ msgid "Could not register ORACLE provider implementation"
#~ msgstr "ORACLE プロバイダの実装を登録することが出来ませんでした"

#~ msgid "postgres_recset is NULL"
#~ msgstr "postgres_recset は NULL です"

#~ msgid "Could not register PostgreSQL provider implementation"
#~ msgstr "PostgreSQL プロバイダの実装を登録することが出来ませんでした"

#~ msgid "Connection seems to be dead"
#~ msgstr "接続は切れたようです"

#~ msgid "sending cmd failed"
#~ msgstr "コマンドの送信に失敗しました"

#~ msgid "Getting recordset userdata failed"
#~ msgstr "レコードセットのユーザデータの取得に失敗しました"

#~ msgid "Command execution finished, all results processed"
#~ msgstr "コマンドの実行は完了しました, すべての結果を処理しました"

#~ msgid "Command failed"
#~ msgstr "コマンドは失敗しました"

#~ msgid "Execution of resultless command succeeded"
#~ msgstr "結果のない実行コマンドは成功しました"

#~ msgid "No further results, command completly succeeded."
#~ msgstr "深刻な結果はありません, コマンドは完全に成功しました"

#~ msgid "Cmd execution returned unknown result, may be an error"
#~ msgstr "コマンド実行は未知の結果を返しました, エラーかもしれません"

#~ msgid "No data returned, freeing recordset structure"
#~ msgstr "データが返されません, レコードセット構造体を開放しています"

#, fuzzy
#~ msgid "Not enough memory for sybase_Connection"
#~ msgstr "primebase_Error 構造体のための十分なメモリがありません"

#~ msgid "schema_cols: table_name '%s'"
#~ msgstr "schema_cols: table_name '%s'"

#~ msgid "Could not allocate enough memory for query"
#~ msgstr "クエリのための十分なメモリを割り当てることが出来ませんでした"

#~ msgid "Could not get recordset in schema_cols"
#~ msgstr "schema_cols からレコードセットを取得することが出来ませんでした"

#~ msgid "schema_procs: invalid constraint type %d"
#~ msgstr "schema_procs: 制約タイプ %d は無効です"

#~ msgid "schema_tables: invalid constraint type %d"
#~ msgstr "schema_tables: 制約タイプ %d は無効です"

#~ msgid "Invalid or unsupported constraint type: %d"
#~ msgstr "制約タイプは無効またはサポートしていません: %d"

#~ msgid "connection status: %x"
#~ msgstr "接続状況: %x"

#~ msgid "return code: %x"
#~ msgstr "戻り値: %x"

#~ msgid "Reconnected to '%s'"
#~ msgstr "'%s' へ再接続しました"

#~ msgid "%s: Could not allocate cmd structure"
#~ msgstr "%s: コマンド構造体を割り当てることが出来ませんでした"

#~ msgid "%s: Could not allocate memory for cmd string"
#~ msgstr "%s: コマンド文字列のためのメモリを割り当てることが出来ませんでした"

#~ msgid "%s: selecting database failed"
#~ msgstr "%s: データベースの選択に失敗しました"

#~ msgid "Could not configure locale to be '%s'"
#~ msgstr "'%s' にロケールを設定することが出来ませんでした"

#~ msgid "Could not drop locale structure"
#~ msgstr "ロケール構造体をドロップすることが出来ませんでした"

#~ msgid "gda_sybase_error_make(%p,%p,%p,%p)"
#~ msgstr "gda_sybase_error_make(%p,%p,%p,%p)"

#~ msgid "No error flag ist set, though %s is called. This may be a bug."
#~ msgstr ""
#~ "%s が呼ばれたにもかかわらずエラーフラグが設定されていません. これは不具合"
#~ "かもしれません"

#~ msgid "cs_diag() initialized"
#~ msgstr "cs_diag() を初期化しました"

#~ msgid "Could not initialize ct_diag()"
#~ msgstr "ct_diag() を初期化することが出来ませんでした"

#~ msgid "%s could not find server implementation"
#~ msgstr "%s をサーバの実装から見付けることが出来ませんでした"

#~ msgid "%s could not get cnc userdata"
#~ msgstr "%s は cnc ユーザデータを取得することが出来ませんでした"

#~ msgid "Could not get # of messages"
#~ msgstr "メッセージ数を取得することが出来ませんでした"

#~ msgid "%d ctlib message to fetch"
#~ msgstr "%d 個の ctlib メッセージを取得しました"

#~ msgid "error is nullpointer in gda_sybase_messages_handle"
#~ msgstr "gda_sybase_messages_handle の NULL ポインタはエラーです"

#~ msgid "Could not get # of server messages"
#~ msgstr "サーバメッセージ数を取得することが出来ませんでした"

#~ msgid "Fetching %d client messages"
#~ msgstr "%d 個のクライアントメッセージを取得しています"

#~ msgid "Fetching %d client message"
#~ msgstr "%d 個のクライアントメッセージを取得しています"

#~ msgid "Could not clear client messages"
#~ msgstr "クライアントメッセージを消去することが出来ませんでした"

#~ msgid "Fetching %d server messages"
#~ msgstr "%d 個のサーバメッセージを取得しています"

#~ msgid "Fetching %d server message"
#~ msgstr "%d 個のサーバメッセージを取得しています"

#~ msgid "Could not get server msg # %d"
#~ msgstr "サーバメッセージ番号 %d を取得することが出来ませんでした"

#~ msgid "Could not get # of cslib messages"
#~ msgstr "cslib メッセージ数を取得することが出来ませんでした"

#~ msgid "Fetching %d cslib messages"
#~ msgstr "%d 個の cslib メッセージを取得してします"

#~ msgid "Fetching %d cslib message"
#~ msgstr "%d 個の cslib メッセージを取得してします"

#~ msgid "Could not get cslib msg # %d"
#~ msgstr "cslib メッセージ番号 %d を取得することが出来ませんでした"

#~ msgid "Could not clear cslib messages"
#~ msgstr "cslib メッセージを消去することが出来ませんでした"

#~ msgid "CS-Library error:"
#~ msgstr "CS-Library エラー:"

#~ msgid "fetch error at row %d"
#~ msgstr "%d 列目の取得でエラーが発生しました"

#~ msgid "CS_END_DATA on row %d; canceling"
#~ msgstr "%d 列目で CS_END_DATA; キャンセル中"

#~ msgid "%s: Error (%d)"
#~ msgstr "%s: エラー (%d)"

#~ msgid "Failed fetching # of columns"
#~ msgstr "カラム数の取得に失敗しました"

#~ msgid "%s could not allocate datafmt info"
#~ msgstr "%s はデータフォーマット情報を割り当てることが出来ませんでした"

#~ msgid "%s could not allocate data fields"
#~ msgstr "%s はデータフィールドを割り当てることが出来ませんでした"

#~ msgid "ct_describe on col # %d"
#~ msgstr "カラム番号 %d で ct_describe します"

#~ msgid "ct_describe on col # %d failed"
#~ msgstr "カラム番号 %d で ct_describe に失敗しました"

#~ msgid "ct_describe(%d): '%s', %d (%s)"
#~ msgstr "ct_describe(%d): '%s', %d (%s)"

#~ msgid "could not ct_bind data holder to column"
#~ msgstr "カラムにデータホルダーを ct_bind することが出来ませんでした"

#~ msgid "illegal type detected, aborting"
#~ msgstr "不当なタイプを検出しました, 中止しています"

#~ msgid "Could not register gda-sybase provider implementation"
#~ msgstr "gda-sybase プロバイダの実装を登録することが出来ませんでした"

#~ msgid "GDA Datasource Access for primebase"
#~ msgstr "primebase用のGDAデータソースアクセス"

#~ msgid "Could not connect to '%s@%s:%s' with connparm '%s'"
#~ msgstr "'%s@%s:%s'へ接続パラメータ '%s'で接続することが出来ませんでした"

#~ msgid "%2ld: Connected to '%s@%s:%s' with connparm: '%s'"
#~ msgstr "%2ld: '%s@%s:%s'へ接続パラメータ '%s'で接続しました"

#~ msgid "Could not get PBIConnectionInfo(), disconnecting"
#~ msgstr "PBIConnectionInfo() を得ることが出来ませんでした, 切断しています"

#~ msgid "%2ld: ConInfo: '%s@%s', Cl v%x, Srv '%s' v%x, Brand '%s'"
#~ msgstr "%2ld: ConInfo: '%s@%s', Cl v%x, Srv '%s' v%x, Brand '%s'"

#~ msgid "Not enough memory for primebase_Error structure."
#~ msgstr "primebase_Error 構造体のための十分なメモリがありません"

#~ msgid "%2ld: ERROR: %ld (%ld) : %s.\n"
#~ msgstr "%2ld: エラー: %ld (%ld) : %s.\n"

#~ msgid "Could not register PRIMEBASE provider implementation"
#~ msgstr "PRIMEBASE プロバイダの実装を登録することが出来ませんでした"

#~ msgid "GDA Datasource Access for Sybase/MS SQL"
#~ msgstr "Sybase/MS SQL用のGDAデータソースアクセス"

#, fuzzy
#~ msgid "Not enough memory for tds_Connection"
#~ msgstr "primebase_Error 構造体のための十分なメモリがありません"

#~ msgid "Reconnected to %s"
#~ msgstr "'%s' へ再接続しました"

#~ msgid "gda_tds_error_make(%p,%p,%p,%p)"
#~ msgstr "gda_tds_error_make(%p,%p,%p,%p)"

#~ msgid "Could not register gda-tds provider implementation"
#~ msgstr "gda-tds プロバイダの実装を登録することが出来ませんでした"

#~ msgid "%s: misconfigured DSN entry"
#~ msgstr "%s: DSN エントリの設定ミスです"

#~ msgid "%s: data source '%s' not found\n"
#~ msgstr "%s: データソース '%s' は見つかりませんでした\n"

#, fuzzy
#~ msgid "Could not register <PROVIDER-NAME> provider implementation"
#~ msgstr "PRIMEBASE プロバイダの実装を登録することが出来ませんでした"

#~ msgid "%s: The server didn't respond."
#~ msgstr "%s: サーバは応答しませんでした"

#~ msgid "thread is already running"
#~ msgstr "スレッドはすでに実行中です"

#~ msgid "impl_GDA_Connection_open: opening connection with DSN: %s"
#~ msgstr "impl_GDA_Connection_open: DSN '%s' で接続を開いています"

#~ msgid "%s: an error ocurred while fetching data"
#~ msgstr "%s: データ取得中にエラーが発生しました"

#~ msgid ""
#~ "OAF doesn't know about our IID; indicates broken installation. Exiting..."
#~ msgstr ""
#~ "OAF は IID について知りません; インストールが失敗しているかもれません. 終"
#~ "了します..."

#~ msgid ""
#~ "Another instance of this provider is already registered with OAF. "
#~ "Exiting..."
#~ msgstr ""
#~ "このプロバイダの別のインスタンスはすでに OAF が登録しています. 終了しま"
#~ "す..."

#~ msgid "Unknown error registering this provider with OAF. Exiting"
#~ msgstr ""
#~ "OAF がこのプロバイダを登録中に未知のエラーが発生しました. 終了します"

#~ msgid "Registered with ID = %s"
#~ msgstr "ID = %s で登録しました"

#~ msgid "CORBA exception unloading report engine: %s"
#~ msgstr "CORBA 例外はレポートエンジンを取り外しています: %s"

#~ msgid "Received signal %d, shutting down."
#~ msgstr "シグナル %d を受信しました, 終了します"

#~ msgid "Received signal %d, shutting down cleanly"
#~ msgstr "シグナル %d を受信しました, きれいに終了します"

#~ msgid "Failed to initialize OAF: please mail bug report to OAF maintainers"
#~ msgstr "OAF の初期化に失敗: OAF メンテナへ不具合報告をメールしてください"

#~ msgid ""
#~ "Another instance of the report engine is already registered with OAF. "
#~ "Exiting..."
#~ msgstr ""
#~ "レポートエンジンの別のインスタンスはすでに OAF が登録しています. 終了しま"
#~ "す..."

#~ msgid "Unknown error registering report engine with OAF. Exiting"
#~ msgstr ""
#~ "OAF がレポートエンジンを登録中に未知のエラーが発生しました. 終了します"

#~ msgid "Registered report engine with ID = %s"
#~ msgstr "ID = %s でレポートエンジンを登録しました"

#~ msgid "%s: length = %d"
#~ msgstr "%s: 長さ = %d"

#~ msgid "%s: sending field description for '%s'"
#~ msgstr "%s: '%s' のためのフィールドの記述を送信中"

#~ msgid "schema_tables: table name = '%s'"
#~ msgstr "schema_tables: テーブル名 = '%s'"

#~ msgid "schema_tables: table_schema = '%s'"
#~ msgstr "schema_tables: table_schema = '%s'"

#~ msgid "schema_procedures: proc_catalog = '%s' UNUSED!\n"
#~ msgstr "schema_procedures: proc_catalog = %s' は使っていません!\n"

#~ msgid "schema_procedures: proc_catalog = '%s' UNUSED!"
#~ msgstr "schema_procedures: proc_catalog = '%s' は使っていません!"

#~ msgid "DATABASE"
#~ msgstr "DATABASE"

#~ msgid "USER-NAME"
#~ msgstr "USER-NAME"

#~ msgid "PASSWORD"
#~ msgstr "PASSWORD"

#~ msgid "FILE-NAME"
#~ msgstr "FILE-NAME"
